demos/security-trifecta/src/pages/login/UnlockCard.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  IonButton,
+  IonButtons,
+  IonCard,
+  IonCardContent,
+  IonCardHeader,
+  IonCardSubtitle,
+  IonCardTitle,
+  IonIcon,
+  IonSelect,
+  IonSelectOption,
+} from '@ionic/react';
+import { useContext, useEffect, useState } from 'react';
+import { logoAmazon, logOut, cafeOutline } from 'ionicons/icons';
+import { useHistory } from 'react-router-dom';
+import { login, logout, isAuthenticated, getAccessToken } from '../../utils/auth';
+import { UnlockMode } from '../../models/UnlockMode';
+import { getUnlockMode, setUnlockMode, canUseLocking } from '../../utils/session-vault';
+import { Device } from '@ionic-enterprise/identity-vault';
+
+const UnlockCard: React.FC = () => {
+  const history = useHistory();
+  const [displayLockOptions, setLockOptions] = useState<boolean>(false);
```

Comment:
This component seems to have a few unused imports and LOC, which can be removed.

Reply:
Done